format/index.md

```diff
@@ -593,10 +594,9 @@ Data, including resources and relationships, can be fetched by sending a
 `GET` request to an endpoint.
 
 Clients **MUST** indicate that they can accept the JSON API media type, per
-the semantics of the HTTP `Accept` header. If present, this header value **MUST**
-also include any media type extensions relevant to the request. Servers **MUST**
-return a `406 Not Acceptable` status code if this header specifies an
-unsupported media type.
+the semantics of the HTTP `Accept` header. Servers **MUST** return a
+`406 Not Acceptable` status code if this header specifies only unsupported
+media types.
```

Comment:
Perhaps it would now be better to cover this in the *Media Type Negotiation* section because it's part of the same logic:

  * If the media type is wrong, return 406 Not Acceptable.
  * If the media type is right but there's a parameter, return 400 Bad Request.

One might also ask if the rules governing Content-Type should work similarly:

  * If the media type is wrong, return 415 Unsupported Media Type.
  * If the media type is right but there's a parameter, return 400 Bad Request.